doc/changelog.rst

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+Changes in Version 4.14.0 (XXXX/XX/XX)
+--------------------------------------
+PyMongo 4.14 brings a number of changes including:
+
+- Added :meth:`bson.codec_options.TypeRegistry.codecs` and :meth:`bson.codec_options.TypeRegistry.fallback_encoder` properties
```

Comment:
Nit, we should use `:attr:`https://www.sphinx-doc.org/en/master/usage/domains/python.html#role-py-attr